app/src/main/java/com/example/easysurvey/ui/theme/MainViewModel.java

```diff
@@ -0,0 +1,25 @@
+package com.example.easysurvey.ui.theme;
+//may need to fix imports
+import android.util.Log
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.example.easysurvey.dao.IEasySurveyDAO;
+import com.example.easysurvey.service.SurveyService;
+import com.google.firebase.firestore.FirebaseFirestore
+import com.google.firebase.firestore.FirebaseFirestoreSettings
+import kotlinx.coroutines.launch
+
+class MainViewModel (var easysurvey:IEasySurveyDAO =SurveyService()) : ViewModel() {
+    val Survey:MutableLiveData<List<Survey>>=MutableLiveData<List<Survey>>()
+
+private lateinit var firestore:FirebaseFirestore
+
+        //only reason not working is because the import is going through for firebase
+        init{
+        firestore=FirebaseFirestore.getInstance()
+        firestore.firestoreSettings=FirebaseFirestoreSettings.Builder().build()
+        }
+
+        //add a fetch function here
+}
```

Comment:
try to add firebase based on the discussed in the lecture videos also koin & Mainviewmodel which is also based on the discussed in the lecture videos

Reply:
This looks like an incomplete feature.
Are you using these anywhere?  I don't see them used.
If not in use, remove them.  Unused code is clutter, and bad documentation, which increases technical debt.
Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.